### ISSUE TITLE
Glass's Delta for effect size

### ISSUE THREAD

**Issue by atredennick** [klapierre/C2E#3]
In commit 5264cd4 I added some code for calculating effect size for different RAC metrics using Glass's Delta. You can find some information on Glass's Delta, and other effect size metrics, here: https://en.wikipedia.org/wiki/Effect_size#Glass'_%CE%94. I think this should work well for our data, except where the standard deviation of the controls is zero. There are also some other errors I haven't completely sorted out, so beware!

The code is [here](https://github.com/klapierre/C2E/blob/master/Community%20Paper/testing_trt_cntl_comps.R).

Let;s keep this issue open until we have sorted out the errors that occur for some data when calculating Glass's d.

**Comment by atredennick** [comment]
@wilcoxkr, did you get this working with no errors?

**Comment by wilcoxkr** [comment]
Yep, it should be working error free. Although I have my working directory in there so you'll need to change it. Do you know of a good generalization that would allow people to skip this step? or someway to automate the path name?
